### PR TITLE
Changing the names of the tables

### PR DESCRIPTION
**Pull request by crufter**

### CODE REVIEW

**Comment by crufter** [comment]
I wonder if we should name things prefixed, like TableMap and TableMultimap to help autocompletion.

**Comment by obeattie** [comment]
Can you detail what this means for users of the library? To what will we need to change our C\* table names?

**Comment by crufter** [comment]
AFAIK no one uses this library in a prod setting, for those who do here it is:

Replace int table names:
- oneToOne -> map
- oneToMany -> multimap
- timeSeries was left intact
- timeseriesB is more problematic because we had a bug in master which named both timeSeries and timeSeriesB identically, although there is one way to differentiate: %s_timeSeries_%s_%s_%s_%s (the one with more underscores in the name) is the timeSeriesB, which now becomes multiTimeSeries

I think we should enum these table types internally so we are more resilient to changes (or leave the table type out completely, since entity name + the partition keys and clustering columns uniquely identify a table).

**Comment by obeattie** [comment]
Ah, I see, sorry I forgot we said there was nothing in prod. I'm an idiot.

**Comment by crufter** [comment]
It was a valid question.

File: README.md [hunk 1]
```diff
@@ -55,24 +55,24 @@ func main() {
...
 
-`OneToOneTable` provides only very simple [CRUD](http://en.wikipedia.org/wiki/Create,_read,_update_and_delete) functionality:
+`MapTable` provides only very simple [CRUD](http://en.wikipedia.org/wiki/Create,_read,_update_and_delete) functionality:
 
 ```go
-    sales := keySpace.OneToOneTable("sale", "Id", Sale{})
+    sales := keySpace.MapTable("sale", "Id", Sale{})
     // …
     result := &Sale{}
     err := sales.Read("sale-1", result)
```
Comment (andreas) [inline README.md:66]: Add `.Run()`?

File: README.md [hunk 2]
```diff
@@ -55,24 +55,24 @@ func main() {
...
 
-`OneToManyTable` can list rows filtered by equality of a single field (eg. list sales based on their `sellerId`):
+`MultimapTable` can list rows filtered by equality of a single field (eg. list sales based on their `sellerId`):
 
 ```go
-    saleTables := keySpace.OneToManyTable("sale", "SellerId", "Id", Sale{})
+    saleTables := keySpace.MultimapTable("sale", "SellerId", "Id", Sale{})
     // …
     results := &[]Sale{}
     err := sales.List("seller-1", nil, 0, results)
```
Comment (andreas) [inline README.md:78]: Add `.Run()`

File: README.md [hunk 3]
```diff
@@ -89,12 +89,12 @@ func main() {
     err := sales.List(yesterdayTime, todayTime, results)
```
Comment (andreas) [inline README.md:89]: Add `.Run()`

File: README.md [hunk 4]
```diff
@@ -89,12 +89,12 @@ func main() {
...
 
-`TimeSeriesBTable` is like a cross between `OneToManyTable` and `TimeSeriesTable`. It can list rows within a time interval, and filtered by equality of a single field. The following lists sales in a time interval, by a certain seller:
+`MultiTimeSeriesTable` is like a cross between `MultimapTable` and `TimeSeriesTable`. It can list rows within a time interval, and filtered by equality of a single field. The following lists sales in a time interval, by a certain seller:
 
 ```go
-    salesTable := keySpace.TimeSeriesBTable("sale", "SellerId", "Created", "Id", Sale{})
+    salesTable := keySpace.MultiTimeSeriesTable("sale", "SellerId", "Created", "Id", Sale{})
     //...
     results := &[]Sale{}
     err := sales.List("seller-1", yesterdayTime, todayTime, results)
```
Comment (andreas) [inline README.md:100]: Add `.Run()`

Comment (andreas) [inline README.md:100]: Also either `sales` or `salesTable`.

Comment (andreas) [inline README.md:100]: (goes for the above as well)

Comment (crufter) [inline README.md:100]: .Run() changes are in master already. Agreed on sales/salesTable consistency, will do it in an other PR, I only made table name change related changes to the Readme here.

**Pull request merged**